src/services/Currency.js

```diff
@@ -0,0 +1,20 @@
+import { feathersClient } from '../lib/feathersClient';
+import { resolve } from 'dns';
+
+class Currency {
+  fiatTypes = ['BRL', 'CAD', 'CHF', 'CZK', 'EUR', 'GBP', 'MXN', 'THB', 'USD'];
+
+  /**
+   * Get fiat type
+   * @param id
+   */
+
+  getFiatType(id) {
```

Comment:
This function isn't used anywhere, I don't understand what it really does and why it is here.